### DISCUSSION TITLE
My Login Page Become Full Width after composer update

### DISCUSSION THREAD

**Discussion by aqilmujahidiin** [filamentphp/filament#15006] [Help]
### Package

Other

### Package Version

v3

### How can we help you?

how to fix this, login page become full width i just done with composer update recently

![image](https://github.com/user-attachments/assets/4abc67cd-6dbe-4741-af6d-22b8a07b2a3d)

thank you for help

**Comment by leandrocfe** [comment]
Any Filament customizations?
If you are using a custom theme, run `npm run dev`

**Comment by aqilmujahidiin** [comment]
no, I didn't do any customization, only after running composer update it became like that, no idea about that, there are other solutions besides npm run dev I have tried it with no effect, thank you

**Comment by leandrocfe** [comment]
Try `php artisan filament:upgrade`

**Comment by xBigDaddyx** [comment]
->simplePageMaxContentWidth(MaxWidth::Small)
try that, documentation is updated.

https://filamentphp.com/docs/3.x/panels/configuration#customizing-the-maximum-content-width

**Comment by aqilmujahidiin** [comment]
the problem is from app.css when i run composer update it broke my css

how to solve this?

thanks

![image](https://github.com/user-attachments/assets/9a21593a-1cc6-4f93-a62e-451008f5db90)

**Reply by leandrocfe** [reply]
Did you run `php artisan filament:upgrade` ?

**Reply by aqilmujahidiin** [reply]
i copied the app.css from my other project and its run correctly, then i run php artisan filament:upgrade so the problem comeback again, my css broken again
